common/rdm/RDMCommand.cpp

```diff
@@ -341,14 +346,18 @@ RDMRequest* RDMRequest::InflateFromData(const uint8_t *data,
   if (code != RDM_COMPLETED_OK)
     return NULL;
 
-  uint16_t sub_device = ((command_message.sub_device[0] << 8) +
-    command_message.sub_device[1]);
-  uint16_t param_id = ((command_message.param_id[0] << 8) +
-    command_message.param_id[1]);
-
+  uint16_t sub_device = JoinUInt8(command_message.sub_device[0],
```

Comment:
The VerifyData() method does most of the work. This does get cleaned up a bit more in the next PR.